### ISSUE TITLE
Calendar is only showing last week for current month.

### ISSUE THREAD

**Issue by jlindenbaum** [prolificinteractive/material-calendarview#74]
I can't seem to reproduce this on my emulators, but on stock Nexus 4 and Nexus 5 both running 5.1.1 I can reproduce this consistently. I can reproduce this on `0.4.0`, `0.5.0`, and `0.6.0`.

The calendar view only shows the last week of the current month, and then shows the greyed out days of next month, scrolling to the next month shows the active day buttons of the last week of the next month, and then greyed out day buttons of the month after that.

For example. Opening up the calendar in June shows, under the month bar, the weekdays from Sunday to Monday. Below I see 26, 27, 28, 29. When I select those I properly get `June 26 2015` as dates. The remaining days in the calendar are not tappable and appear to be for July.

When I swipe over to August it again shows me 26, 27, 28, 29, 30, 31 along the top - which, when tapped, are the correct `July 26 2015` dates. The remainder of the days appear to be August, and are not selectable.

The fragment view code is below, as well as a screen shot of the issue.

I am using DayDecorators.

I build against 21, with 21.1.1 build tools. Support library `support-v13:21.0.3` with app compat `appcompat-v7:22.2.0`

```
<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
                xmlns:app="http://schemas.android.com/apk/res-auto"
                android:layout_width="fill_parent"
                android:layout_height="fill_parent"
                android:background="@color/screen_background_light"
                android:orientation="vertical">

    <com.prolificinteractive.materialcalendarview.MaterialCalendarView
        android:id="@+id/event_calendar_view"
        android:layout_width="match_parent"
        android:layout_height="320dp"
        app:mcv_showOtherDates="true"
        app:mcv_tileSize="40dp"/>

    <include
        android:id="@+id/list_header_title"
        layout="@layout/list_header"
        android:layout_width="match_parent"
        android:layout_height="30dip"
        android:layout_below="@+id/event_calendar_view"/>

    <ProgressBar
        android:id="@+id/event_progress_bar"
        android:layout_width="50dip"
        android:layout_height="50dip"
        android:layout_below="@id/list_header_title"
        android:layout_centerHorizontal="true"
        android:layout_marginTop="20dip"/>

    <ListView
        android:id="@+id/event_list_view"
        android:layout_width="match_parent"
        android:layout_height="fill_parent"
        android:layout_below="@+id/list_header_title"
        android:visibility="gone"/>
</RelativeLayout>
```

![screenshot_2015-07-02-12-50-51](https://cloud.githubusercontent.com/assets/197034/8485525/69fa381e-20bb-11e5-9727-e9e2a6e0247b.png)

**Comment by dandc87** [comment]
Hey @jlindenbaum, I wasn't able to replicate this on any of my test devices (including a Nexus 5). Do you see this issue with the sample app as well?

**Comment by jlindenbaum** [comment]
I can't reproduce it on the sample app. Played around with the sample app's XML, too, to see if it's something in there.

I stripped out my fragment to just the calendar view, removed all decorators, and set options and other views on the screen, and it still does it. Still the same results.

**Comment by jlindenbaum** [comment]
I can reproduce this across all my test devices, including some 4.x devices, but not on the simulators. I'm upgrading from `0.2.4` - I've stripped everything in that fragment, and just a bare-bones setup of the calendar view is rendering like that on `0.6.1`. `0.2.4` works as expected.

**Comment by gusdn9** [comment]
I have same issue. testing on nexus5, galaxy s5.
![screenshot_2015-07-06-10-25-42](https://cloud.githubusercontent.com/assets/3655998/8514392/baa793c2-23c9-11e5-83b3-87b561dcecbc.png)
![screenshot_2015-07-06-10-25-46](https://cloud.githubusercontent.com/assets/3655998/8514393/bdf17c3c-23c9-11e5-823e-c153ccbb4e85.png)

**Comment by sersast** [comment]
same problem but I`ve explored a little bit further. It seems it depends on the TimeZone.
With the following configuration:
![settings_not_working](https://cloud.githubusercontent.com/assets/13233962/8566136/37edfbee-255c-11e5-9f91-9d7066bafc10.png)
I reproduce this issue as shown in the next image:
![calendar_not_working](https://cloud.githubusercontent.com/assets/13233962/8566155/54208e62-255c-11e5-8497-3e6cb98f161a.png)
Then if I modify my settings, concretely the time zone as follows:
![settings_working](https://cloud.githubusercontent.com/assets/13233962/8566167/749fc7d4-255c-11e5-901b-8deac325a066.png)
The calendar is displayed correctly, as depicted in the following pic:
![calendar_working](https://cloud.githubusercontent.com/assets/13233962/8566175/8c05524a-255c-11e5-9ece-54363253bc33.png)
Hope it helps!

**Comment by dandc87** [comment]
@sersast thanks for the research!

**Comment by dandc87** [comment]
Fixed in 0.7.0
